Source/WebKitLegacy/mac/WebView/WebHTMLView.mm

```diff
@@ -590,28 +587,23 @@ - (void)forwardContextMenuAction:(id)sender;
         return WebMenuItemTagDictationAlternative;
     case ContextMenuItemTagToggleVideoFullscreen:
         return WebMenuItemTagToggleVideoFullscreen;
-    case ContextMenuItemTagAddHighlightToCurrentQuickNote:
-    case ContextMenuItemTagAddHighlightToNewQuickNote:
-        return std::nullopt;
     case ContextMenuItemTagShareMenu:
         return WebMenuItemTagShareMenu;
     case ContextMenuItemTagToggleVideoEnhancedFullscreen:
         return WebMenuItemTagToggleVideoEnhancedFullscreen;
     case ContextMenuItemTagTranslate:
         return WebMenuItemTagTranslate;
-    case ContextMenuItemTagCopySubject:
-    case ContextMenuItemTagLookUpImage:
-    case ContextMenuItemPDFSinglePageContinuous:
-    case ContextMenuItemPDFTwoPages:
-    case ContextMenuItemPDFTwoPagesContinuous:
-        return std::nullopt;
 
     case ContextMenuItemBaseCustomTag ... ContextMenuItemLastCustomTag:
         // We just pass these through.
         return static_cast<NSInteger>(action);
 
     case ContextMenuItemBaseApplicationTag:
         ASSERT_NOT_REACHED();
+        break;
+
+    default:
+        break;
```

Comment:
both cause i need to add a case for `ContextMenuItemTagShowMediaStats` and so that in the future if others add new items like that they don't even have to worry about handling it here